### PR TITLE
Readme sync

### PR DESCRIPTION
**Pull request by kuzdogan**
Fetches the README content of each Sourcify service from GitHub and renders them to "Running Sourcify" page of each service.

### CODE REVIEW

File: docs/2. Running Sourcify/1-Server.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+slug: /running-server
+---
+
+import EmbedReadme from "./EmbedReadme";
+const URL = "https://raw.githubusercontent.com/ethereum/sourcify/master/services/server/README.md";
+const displayURL = URL.replace("https://raw.githubusercontent.com/", "");
+export const ReadmeLink = () => (<a href={URL}>{displayURL}</a>);
+
+*Content from <ReadmeLink/>*
+<EmbedReadme url={URL}/>
```
Comment (marcocastignoli) [inline docs/2. Running Sourcify/1-Server.mdx:12]: I like this solution 👍

**Review by marcocastignoli — APPROVED** [review]
I didn't try it, did you check if you have an extension that allow cors? also I know firefox block fetched 3rd party content on websites. Did you try?

**Comment by kuzdogan** [comment]
> I didn't try it, did you check if you have an extension that allow cors? also I know firefox block fetched 3rd party content on websites. Did you try?

Works on Firefox. 

CORS should work fine for raw.githubusercontent.com

**Comment by kuzdogan** [comment]
> I didn't try it, did you check if you have an extension that allow cors? also I know firefox block fetched 3rd party content on websites. Did you try?

Works on Firefox. 

CORS should work fine for raw.githubusercontent.com. 3rd party "cookies" are different than "content. raw.githubusercontent.com should have `-Allow-Origin: *` to enable all access.

**Pull request merged**